api/app/queue_config.py

```diff
@@ -0,0 +1,68 @@
+import os
+import json
+
+
+def extract_predefined_queues():
+    url = os.environ.get('SQS_QUEUE_URL', '')
+    key = os.environ.get('SQS_AWS_ACCESS_KEY_ID', '')
+    secret = os.environ.get('SQS_AWS_SECRET_ACCESS_KEY', '')
+    if url and key and secret:
```

Comment:
Am I reading this right that we prefer local env variables to VCAP?

The way we do our other VCAP stuff is to export vars over the top of existing environment thereby taking precedence. This suggests the inverse is happening here

Reply:
You read it right. I will change it so that it applies VCAP values first and then overwrites if the single queue env config exists. While we do use these env as "local", there's nothing technically defining those as only for local—prod could just as well use those env vars if it made more sense to have a single queue in prod too.

Reply:
Actually let's chat about this.